bin/fissile

```diff
@@ -10,12 +10,14 @@ if [ -f ${FISSILE_GIT_ROOT}/Jenkinsfile ] ; then
     # add the common part, which references them.
 
     if [ "${USE_SLE_BASE}" = "false" ]; then
-	cp "${FISSILE_LIGHT_OPEN42}" "${FISSILE_LIGHT_OPINIONS}"
+        SRC_BASE="${FISSILE_LIGHT_OPEN42}"
     else
-	cp "${FISSILE_LIGHT_SLE12}" "${FISSILE_LIGHT_OPINIONS}"
+        SRC_BASE="${FISSILE_LIGHT_SLE12}"
     fi
 
-    cat "${FISSILE_LIGHT_COMMON}" >> "${FISSILE_LIGHT_OPINIONS}"
+    tmp_file=$(mktemp -t tmp.XXXXXXXXXXXXXXXXXXXX) || exit 1
```

Comment:
Huh, we don't have `set -o errexit` on this script? This means that, e.g., the `cat` could fail silently…

Reply:
With `set -o errexit` you no longer need `|| exit 1`, but it's harmless